[32mINFO    [0m juju.model:model.py:2971 Waiting for model:
  shard-one/0 [idle] active: Primary
  shard-one/1 [idle] active: 
  shard-one/2 [idle] active: 
  shard-two/0 [idle] active: Primary
  shard-two/1 [idle] active: 
  shard-two/2 [idle] active: 
  config-server/0 [idle] active: 
  config-server/1 [idle] active: 
  config-server/2 [idle] active:
[32mINFO    [0m pytest_operator.plugin:plugin.py:903 Model status:

Model  Controller           Cloud/Region         Version  SLA          Timestamp
test   localhost-localhost  localhost/localhost  3.5.3    unsupported  03:10:49Z

App            Version  Status  Scale  Charm    Channel  Rev  Exposed  Message
config-server           active      3  mongodb             0  no       
shard-one               active      3  mongodb             1  no       
shard-two               active      3  mongodb             2  no       

Unit              Workload  Agent  Machine  Public address  Ports            Message
config-server/0*  active    idle   0        10.23.166.123   27017-27018/tcp  
config-server/1   active    idle   1        10.23.166.107   27017-27018/tcp  
config-server/2   active    idle   2        10.23.166.91    27017-27018/tcp  
shard-one/0       active    idle   3        10.23.166.193   27017/tcp        Primary
shard-one/1       active    idle   4        10.23.166.21    27017/tcp        
shard-one/2*      active    idle   5        10.23.166.24    27017/tcp        
shard-two/0       active    idle   6        10.23.166.131   27017/tcp        Primary
shard-two/1       active    idle   7        10.23.166.92    27017/tcp        
shard-two/2*      active    idle   8        10.23.166.178   27017/tcp        

Machine  State    Address        Inst id        Base          AZ  Message
0        started  10.23.166.123  juju-a7a3f7-0  ubuntu@22.04      Running
1        started  10.23.166.107  juju-a7a3f7-1  ubuntu@22.04      Running
2        started  10.23.166.91   juju-a7a3f7-2  ubuntu@22.04      Running
3        started  10.23.166.193  juju-a7a3f7-3  ubuntu@22.04      Running
4        started  10.23.166.21   juju-a7a3f7-4  ubuntu@22.04      Running
5        started  10.23.166.24   juju-a7a3f7-5  ubuntu@22.04      Running
6        started  10.23.166.131  juju-a7a3f7-6  ubuntu@22.04      Running
7        started  10.23.166.92   juju-a7a3f7-7  ubuntu@22.04      Running
8        started  10.23.166.178  juju-a7a3f7-8  ubuntu@22.04      Running

[32mINFO    [0m pytest_operator.plugin:plugin.py:909 Juju error logs:

machine-0: 02:43:20 ERROR juju.worker.dependency "lxd-container-provisioner" manifold worker returned unexpected error: container types not yet available
machine-0: 02:43:20 ERROR juju.worker.dependency "kvm-container-provisioner" manifold worker returned unexpected error: container types not yet available
unit-config-server-0: 02:43:21 ERROR juju.worker.meterstatus error running "meter-status-changed": charm missing from disk
machine-2: 02:43:23 ERROR juju.worker.dependency "lxd-container-provisioner" manifold worker returned unexpected error: container types not yet available
machine-2: 02:43:23 ERROR juju.worker.dependency "kvm-container-provisioner" manifold worker returned unexpected error: container types not yet available
unit-config-server-2: 02:43:23 ERROR juju.worker.meterstatus error running "meter-status-changed": charm missing from disk
machine-1: 02:43:24 ERROR juju.worker.dependency "lxd-container-provisioner" manifold worker returned unexpected error: container types not yet available
machine-1: 02:43:24 ERROR juju.worker.dependency "kvm-container-provisioner" manifold worker returned unexpected error: container types not yet available
machine-5: 02:43:24 ERROR juju.worker.dependency "kvm-container-provisioner" manifold worker returned unexpected error: container types not yet available
machine-5: 02:43:24 ERROR juju.worker.dependency "lxd-container-provisioner" manifold worker returned unexpected error: container types not yet available
unit-config-server-1: 02:43:24 ERROR juju.worker.meterstatus error running "meter-status-changed": charm missing from disk
unit-shard-one-2: 02:43:24 ERROR juju.worker.meterstatus error running "meter-status-changed": charm missing from disk
machine-3: 02:43:25 ERROR juju.worker.dependency "lxd-container-provisioner" manifold worker returned unexpected error: container types not yet available
machine-4: 02:43:25 ERROR juju.worker.dependency "lxd-container-provisioner" manifold worker returned unexpected error: container types not yet available
machine-4: 02:43:25 ERROR juju.worker.dependency "kvm-container-provisioner" manifold worker returned unexpected error: container types not yet available
machine-3: 02:43:25 ERROR juju.worker.dependency "kvm-container-provisioner" manifold worker returned unexpected error: container types not yet available
machine-6: 02:43:25 ERROR juju.worker.dependency "lxd-container-provisioner" manifold worker returned unexpected error: container types not yet available
machine-6: 02:43:25 ERROR juju.worker.dependency "kvm-container-provisioner" manifold worker returned unexpected error: container types not yet available
machine-8: 02:43:26 ERROR juju.worker.dependency "kvm-container-provisioner" manifold worker returned unexpected error: container types not yet available
machine-8: 02:43:26 ERROR juju.worker.dependency "lxd-container-provisioner" manifold worker returned unexpected error: container types not yet available
unit-shard-two-2: 02:43:27 ERROR juju.worker.meterstatus error running "meter-status-changed": charm missing from disk
unit-shard-one-0: 02:43:29 ERROR juju.worker.meterstatus error running "meter-status-changed": charm missing from disk
machine-7: 02:43:29 ERROR juju.worker.dependency "lxd-container-provisioner" manifold worker returned unexpected error: container types not yet available
machine-7: 02:43:29 ERROR juju.worker.dependency "kvm-container-provisioner" manifold worker returned unexpected error: container types not yet available
unit-shard-two-0: 02:43:29 ERROR juju.worker.meterstatus error running "meter-status-changed": charm missing from disk
unit-shard-two-1: 02:43:29 ERROR juju.worker.meterstatus error running "meter-status-changed": charm missing from disk
unit-shard-one-1: 02:43:29 ERROR juju.worker.meterstatus error running "meter-status-changed": charm missing from disk
unit-config-server-0: 02:45:00 ERROR unit.config-server/0.juju-log _on_secret_remove: Secret secret:cs68b3l469p1s4f6cnog seems to have no observers, could be removed
unit-shard-two-2: 02:45:10 ERROR unit.shard-two/2.juju-log _on_secret_remove: Secret secret:cs68b5l469p1s4f6cnp0 seems to have no observers, could be removed
unit-shard-one-2: 02:45:10 ERROR unit.shard-one/2.juju-log _on_secret_remove: Secret secret:cs68b5l469p1s4f6cnpg seems to have no observers, could be removed
unit-config-server-0: 02:46:28 ERROR unit.config-server/0.juju-log config-server:6: Failed to add shard shard-one to the config server, error=OperationFailure("Authentication failed., full error: {'ok': 0.0, 'errmsg': 'Authentication failed.', 'code': 18, 'codeName': 'AuthenticationFailed', '$clusterTime': {'clusterTime': Timestamp(1728873987, 2), 'signature': {'hash': b'0\\x96\\xdbc\\xfd\\xcb\\x0c\\x80C7\\xe4<\\x1a\\x03\\x1d2\\xb8\\xd0G\\xf7', 'keyId': 7425456777803595801}}, 'operationTime': Timestamp(1728873987, 2)}")
unit-config-server-0: 02:46:28 ERROR unit.config-server/0.juju-log config-server:6: shard-one shard does not have the same auth as the config server.
unit-config-server-0: 02:46:31 ERROR unit.config-server/0.juju-log config-server:6: Failed to add shard shard-one to the config server, error=OperationFailure("Authentication failed., full error: {'ok': 0.0, 'errmsg': 'Authentication failed.', 'code': 18, 'codeName': 'AuthenticationFailed', '$clusterTime': {'clusterTime': Timestamp(1728873989, 3), 'signature': {'hash': b'\\xf9)+\\xc7\\x13\\xf7\\xa1d\\x16\\x1d\\xfb\\x91;\\xea\\xa7\\x84!\\x16O\\xc4', 'keyId': 7425456777803595801}}, 'operationTime': Timestamp(1728873989, 3)}")
unit-config-server-0: 02:46:31 ERROR unit.config-server/0.juju-log config-server:6: shard-one shard does not have the same auth as the config server.
unit-config-server-0: 02:46:48 ERROR unit.config-server/0.juju-log config-server:6: Failed to add shard shard-one to the config server, error=OperationFailure('Could not find host matching read preference { mode: "primary" } for set shard-one, full error: {\'ok\': 0.0, \'errmsg\': \'Could not find host matching read preference { mode: "primary" } for set shard-one\', \'code\': 133, \'codeName\': \'FailedToSatisfyReadPreference\', \'$clusterTime\': {\'clusterTime\': Timestamp(1728874007, 2), \'signature\': {\'hash\': b\'Deq\\xd8\\xa9\\x8a\\x84(\\xd3H\\x07k\\xab\\xce\\x85Z\\xb0\\x15`\\x95\', \'keyId\': 7425456777803595801}}, \'operationTime\': Timestamp(1728874007, 2)}')
unit-config-server-0: 02:46:48 ERROR unit.config-server/0.juju-log config-server:6: Failed to add shard-one to cluster
unit-config-server-0: 02:46:48 ERROR unit.config-server/0.juju-log config-server:6: Deferring _on_relation_event for shards interface since: error=OperationFailure('Could not find host matching read preference { mode: "primary" } for set shard-one, full error: {\'ok\': 0.0, \'errmsg\': \'Could not find host matching read preference { mode: "primary" } for set shard-one\', \'code\': 133, \'codeName\': \'FailedToSatisfyReadPreference\', \'$clusterTime\': {\'clusterTime\': Timestamp(1728874007, 2), \'signature\': {\'hash\': b\'Deq\\xd8\\xa9\\x8a\\x84(\\xd3H\\x07k\\xab\\xce\\x85Z\\xb0\\x15`\\x95\', \'keyId\': 7425456777803595801}}, \'operationTime\': Timestamp(1728874007, 2)}')
unit-config-server-0: 02:47:13 ERROR unit.config-server/0.juju-log config-server:6: Failed to add shard shard-two to the config server, error=OperationFailure("Authentication failed., full error: {'ok': 0.0, 'errmsg': 'Authentication failed.', 'code': 18, 'codeName': 'AuthenticationFailed', '$clusterTime': {'clusterTime': Timestamp(1728874033, 2), 'signature': {'hash': b'7\\x96\\xa6\\xc1\\xf0H8\\xc3\\xebh\\x05\\xe9y\\x1d\\x07\\xf2f-[\\xcc', 'keyId': 7425456777803595801}}, 'operationTime': Timestamp(1728874033, 2)}")
unit-config-server-0: 02:47:13 ERROR unit.config-server/0.juju-log config-server:6: shard-two shard does not have the same auth as the config server.
unit-shard-one-2: 02:47:13 ERROR unit.shard-one/2.juju-log _on_secret_remove: Secret secret:cs68b5l469p1s4f6cnpg seems to have no observers, could be removed
unit-shard-one-2: 02:47:21 ERROR unit.shard-one/2.juju-log _on_secret_remove: Secret secret:cs68b5l469p1s4f6cnpg seems to have no observers, could be removed
unit-shard-one-2: 02:47:27 ERROR unit.shard-one/2.juju-log _on_secret_remove: Secret secret:cs68b5l469p1s4f6cnpg seems to have no observers, could be removed
unit-config-server-0: 02:47:30 ERROR unit.config-server/0.juju-log config-server:7: Failed to add shard shard-two to the config server, error=OperationFailure('Could not find host matching read preference { mode: "primary" } for set shard-two, full error: {\'ok\': 0.0, \'errmsg\': \'Could not find host matching read preference { mode: "primary" } for set shard-two\', \'code\': 133, \'codeName\': \'FailedToSatisfyReadPreference\', \'$clusterTime\': {\'clusterTime\': Timestamp(1728874050, 1), \'signature\': {\'hash\': b\'29}\\x0e\\xf5\\xb1J\\xb2{\\xac\\xe75\\xed\\xe3[>\\x12\\x90(`\', \'keyId\': 7425456777803595801}}, \'operationTime\': Timestamp(1728874050, 1)}')
unit-config-server-0: 02:47:30 ERROR unit.config-server/0.juju-log config-server:7: Failed to add shard-two to cluster
unit-config-server-0: 02:47:30 ERROR unit.config-server/0.juju-log config-server:7: Deferring _on_relation_event for shards interface since: error=OperationFailure('Could not find host matching read preference { mode: "primary" } for set shard-two, full error: {\'ok\': 0.0, \'errmsg\': \'Could not find host matching read preference { mode: "primary" } for set shard-two\', \'code\': 133, \'codeName\': \'FailedToSatisfyReadPreference\', \'$clusterTime\': {\'clusterTime\': Timestamp(1728874050, 1), \'signature\': {\'hash\': b\'29}\\x0e\\xf5\\xb1J\\xb2{\\xac\\xe75\\xed\\xe3[>\\x12\\x90(`\', \'keyId\': 7425456777803595801}}, \'operationTime\': Timestamp(1728874050, 1)}')
unit-shard-one-2: 02:47:32 ERROR unit.shard-one/2.juju-log _on_secret_remove: Secret secret:cs68b5l469p1s4f6cnpg seems to have no observers, could be removed
unit-shard-two-2: 02:48:00 ERROR unit.shard-two/2.juju-log _on_secret_remove: Secret secret:cs68b5l469p1s4f6cnp0 seems to have no observers, could be removed
unit-shard-two-2: 02:48:05 ERROR unit.shard-two/2.juju-log _on_secret_remove: Secret secret:cs68b5l469p1s4f6cnp0 seems to have no observers, could be removed
unit-shard-one-2: 02:48:05 ERROR unit.shard-one/2.juju-log _on_secret_remove: Secret secret:cs68b5l469p1s4f6cnpg seems to have no observers, could be removed
unit-shard-one-2: 02:48:52 ERROR unit.shard-one/2.juju-log _on_secret_remove: Secret secret:cs68b5l469p1s4f6cnpg seems to have no observers, could be removed
unit-shard-two-2: 02:53:51 ERROR unit.shard-two/2.juju-log _on_secret_remove: Secret secret:cs68b5l469p1s4f6cnp0 seems to have no observers, could be removed
unit-shard-one-2: 03:00:05 ERROR unit.shard-one/2.juju-log _on_secret_remove: Secret secret:cs68b5l469p1s4f6cnpg seems to have no observers, could be removed
unit-shard-one-2: 03:00:12 ERROR unit.shard-one/2.juju-log _on_secret_remove: Secret secret:cs68b5l469p1s4f6cnpg seems to have no observers, could be removed
unit-shard-two-2: 03:03:29 ERROR unit.shard-two/2.juju-log _on_secret_remove: Secret secret:cs68b5l469p1s4f6cnp0 seems to have no observers, could be removed
unit-shard-two-2: 03:05:37 ERROR unit.shard-two/2.juju-log _on_secret_remove: Secret secret:cs68b5l469p1s4f6cnp0 seems to have no observers, could be removed
unit-shard-one-2: 03:05:37 ERROR unit.shard-one/2.juju-log _on_secret_remove: Secret secret:cs68b5l469p1s4f6cnpg seems to have no observers, could be removed
unit-shard-two-2: 03:05:38 ERROR unit.shard-two/2.juju-log _on_secret_remove: Secret secret:cs68b5l469p1s4f6cnp0 seems to have no observers, could be removed
unit-config-server-0: 03:09:29 ERROR unit.config-server/0.juju-log Cannot proceed with refresh. Failed to check cluster health, error: No replica set members match selector "Primary()", Timeout: 1.0s, Topology Description: <TopologyDescription id: 670c8b683c70a20ce56c760b, topology_type: ReplicaSetNoPrimary, servers: [<ServerDescription ('10.23.166.193', 27017) server_type: Unknown, rtt: None>, <ServerDescription ('10.23.166.21', 27017) server_type: RSSecondary, rtt: 0.0008732440001040231>, <ServerDescription ('10.23.166.24', 27017) server_type: RSSecondary, rtt: 0.0008752840003580786>]>
unit-config-server-0: 03:09:33 ERROR unit.config-server/0.juju-log Cannot proceed with refresh. Failed to check cluster health, error: No replica set members match selector "Primary()", Timeout: 1.0s, Topology Description: <TopologyDescription id: 670c8b6b3c70a20ce56c7610, topology_type: ReplicaSetNoPrimary, servers: [<ServerDescription ('10.23.166.193', 27017) server_type: Unknown, rtt: None>, <ServerDescription ('10.23.166.21', 27017) server_type: RSSecondary, rtt: 0.0009937449999597447>, <ServerDescription ('10.23.166.24', 27017) server_type: RSSecondary, rtt: 0.000951445000282547>]>
unit-config-server-0: 03:09:36 ERROR unit.config-server/0.juju-log Cannot proceed with refresh. Failed to check cluster health, error: No replica set members match selector "Primary()", Timeout: 1.0s, Topology Description: <TopologyDescription id: 670c8b6f3c70a20ce56c7615, topology_type: ReplicaSetNoPrimary, servers: [<ServerDescription ('10.23.166.193', 27017) server_type: Unknown, rtt: None>, <ServerDescription ('10.23.166.21', 27017) server_type: RSSecondary, rtt: 0.0009247606000826636>, <ServerDescription ('10.23.166.24', 27017) server_type: RSSecondary, rtt: 0.0008546340000066266>]>
unit-shard-one-0: 03:09:59 ERROR juju.worker.dependency "api-caller" manifold worker returned unexpected error: api connection broken unexpectedly
unit-shard-one-0: 03:10:02 ERROR juju.worker.dependency "api-caller" manifold worker returned unexpected error: [e56afd] "unit-shard-one-0" cannot open api: unable to connect to API: dial tcp 10.23.166.10:17070: connect: network is unreachable
unit-config-server-0: 03:10:02 ERROR unit.config-server/0.juju-log Uncaught exception while in charm code:
Traceback (most recent call last):
  File "/var/lib/juju/agents/unit-config-server-0/charm/./src/charm.py", line 1598, in <module>
    main(MongodbOperatorCharm)
  File "/var/lib/juju/agents/unit-config-server-0/charm/venv/ops/main.py", line 551, in main
    manager.run()
  File "/var/lib/juju/agents/unit-config-server-0/charm/venv/ops/main.py", line 530, in run
    self._emit()
  File "/var/lib/juju/agents/unit-config-server-0/charm/venv/ops/main.py", line 519, in _emit
    _emit_charm_event(self.charm, self.dispatcher.event_name)
  File "/var/lib/juju/agents/unit-config-server-0/charm/venv/ops/main.py", line 147, in _emit_charm_event
    event_to_emit.emit(*args, **kwargs)
  File "/var/lib/juju/agents/unit-config-server-0/charm/venv/ops/framework.py", line 348, in emit
    framework._emit(event)
  File "/var/lib/juju/agents/unit-config-server-0/charm/venv/ops/framework.py", line 860, in _emit
    self._reemit(event_path)
  File "/var/lib/juju/agents/unit-config-server-0/charm/venv/ops/framework.py", line 950, in _reemit
    custom_handler(event)
  File "/var/lib/juju/agents/unit-config-server-0/charm/src/upgrades/mongodb_upgrade.py", line 148, in _on_pre_upgrade_check_action
    self._upgrade.pre_upgrade_check()
  File "/var/lib/juju/agents/unit-config-server-0/charm/lib/charms/mongodb/v0/upgrade_helpers.py", line 336, in pre_upgrade_check
    if not self._charm.upgrade.is_cluster_able_to_read_write():
  File "/var/lib/juju/agents/unit-config-server-0/charm/lib/charms/mongodb/v0/upgrade_helpers.py", line 523, in is_cluster_able_to_read_write
    return self.is_sharded_cluster_able_to_read_write()
  File "/var/lib/juju/agents/unit-config-server-0/charm/lib/charms/mongodb/v0/upgrade_helpers.py", line 539, in is_sharded_cluster_able_to_read_write
    write_replicated = self.is_write_on_secondaries(
  File "/var/lib/juju/agents/unit-config-server-0/charm/lib/charms/mongodb/v0/upgrade_helpers.py", line 651, in is_write_on_secondaries
    self.confirm_excepted_write_on_replica(
  File "/var/lib/juju/agents/unit-config-server-0/charm/venv/tenacity/__init__.py", line 336, in wrapped_f
    return copy(f, *args, **kw)
  File "/var/lib/juju/agents/unit-config-server-0/charm/venv/tenacity/__init__.py", line 475, in __call__
    do = self.iter(retry_state=retry_state)
  File "/var/lib/juju/agents/unit-config-server-0/charm/venv/tenacity/__init__.py", line 376, in iter
    result = action(retry_state)
  File "/var/lib/juju/agents/unit-config-server-0/charm/venv/tenacity/__init__.py", line 418, in exc_check
    raise retry_exc.reraise()
  File "/var/lib/juju/agents/unit-config-server-0/charm/venv/tenacity/__init__.py", line 185, in reraise
    raise self.last_attempt.result()
  File "/usr/lib/python3.10/concurrent/futures/_base.py", line 451, in result
    return self.__get_result()
  File "/usr/lib/python3.10/concurrent/futures/_base.py", line 403, in __get_result
    raise self._exception
  File "/var/lib/juju/agents/unit-config-server-0/charm/venv/tenacity/__init__.py", line 478, in __call__
    result = fn(*args, **kwargs)
  File "/var/lib/juju/agents/unit-config-server-0/charm/lib/charms/mongodb/v0/upgrade_helpers.py", line 610, in confirm_excepted_write_on_replica
    if query[0][WRITE_KEY] != expected_write_value:
  File "/var/lib/juju/agents/unit-config-server-0/charm/venv/pymongo/synchronous/cursor.py", line 638, in __getitem__
    for doc in clone:  # type: ignore[attr-defined]
  File "/var/lib/juju/agents/unit-config-server-0/charm/venv/pymongo/synchronous/cursor.py", line 1281, in __next__
    return self.next()
  File "/var/lib/juju/agents/unit-config-server-0/charm/venv/pymongo/synchronous/cursor.py", line 1257, in next
    if len(self._data) or self._refresh():
  File "/var/lib/juju/agents/unit-config-server-0/charm/venv/pymongo/synchronous/cursor.py", line 1205, in _refresh
    self._send_message(q)
  File "/var/lib/juju/agents/unit-config-server-0/charm/venv/pymongo/synchronous/cursor.py", line 1100, in _send_message
    response = client._run_operation(
  File "/var/lib/juju/agents/unit-config-server-0/charm/venv/pymongo/_csot.py", line 119, in csot_wrapper
    return func(self, *args, **kwargs)
  File "/var/lib/juju/agents/unit-config-server-0/charm/venv/pymongo/synchronous/mongo_client.py", line 1754, in _run_operation
    return self._retryable_read(
  File "/var/lib/juju/agents/unit-config-server-0/charm/venv/pymongo/synchronous/mongo_client.py", line 1863, in _retryable_read
    return self._retry_internal(
  File "/var/lib/juju/agents/unit-config-server-0/charm/venv/pymongo/_csot.py", line 119, in csot_wrapper
    return func(self, *args, **kwargs)
  File "/var/lib/juju/agents/unit-config-server-0/charm/venv/pymongo/synchronous/mongo_client.py", line 1830, in _retry_internal
    ).run()
  File "/var/lib/juju/agents/unit-config-server-0/charm/venv/pymongo/synchronous/mongo_client.py", line 2554, in run
    return self._read() if self._is_read else self._write()
  File "/var/lib/juju/agents/unit-config-server-0/charm/venv/pymongo/synchronous/mongo_client.py", line 2689, in _read
    self._server = self._get_server()
  File "/var/lib/juju/agents/unit-config-server-0/charm/venv/pymongo/synchronous/mongo_client.py", line 2645, in _get_server
    return self._client._select_server(
  File "/var/lib/juju/agents/unit-config-server-0/charm/venv/pymongo/synchronous/mongo_client.py", line 1649, in _select_server
    server = topology.select_server(
  File "/var/lib/juju/agents/unit-config-server-0/charm/venv/pymongo/synchronous/topology.py", line 398, in select_server
    server = self._select_server(
  File "/var/lib/juju/agents/unit-config-server-0/charm/venv/pymongo/synchronous/topology.py", line 376, in _select_server
    servers = self.select_servers(
  File "/var/lib/juju/agents/unit-config-server-0/charm/venv/pymongo/synchronous/topology.py", line 283, in select_servers
    server_descriptions = self._select_servers_loop(
  File "/var/lib/juju/agents/unit-config-server-0/charm/venv/pymongo/synchronous/topology.py", line 333, in _select_servers_loop
    raise ServerSelectionTimeoutError(
pymongo.errors.ServerSelectionTimeoutError: No servers found yet, Timeout: 1.0s, Topology Description: <TopologyDescription id: 670c8b893c70a20ce56c7629, topology_type: Single, servers: [<ServerDescription ('10.23.166.193', 27017) server_type: Unknown, rtt: None>]>

[32mINFO    [0m pytest_operator.plugin:plugin.py:991 Forgetting model main...